google-cloud-clients/google-cloud-contrib/google-cloud-logging-logback/README.md

```diff
@@ -88,6 +88,23 @@ Authentication
 See the [Authentication](https://github.com/googleapis/google-cloud-java#authentication)
 section in the base directory's README.
 
+You can also specify custom credentials and other options by creating a subclass of `com.google.cloud.logging.logback.LoggingAppender` and override the method `createLoggingOptions()`. Your logback.xml configuration file must reference your subclass instead of `com.google.cloud.logging.logback.LoggingAppender`.
+
```

Comment:
do we still need these instructions for overriding createLoggingOptions?

Reply:
You're right, I changed these instructions into using the new custom property fileCredentials.